BBPebbleLib/Pebble.cpp

```diff
@@ -38,46 +44,88 @@ void Pebble::onDisconnect()
 
 void Pebble::onDataReceived(QBluetoothSocket &bt_socket)
```

Comment:
I had issues before with the received data (the length and debugged output were wrong) so i took the more widely tested implementation to avoid communication issues